projects/wave-core/src/lib/operators/dialogs/data-repository/data-repository.component.scss

```diff
@@ -16,6 +16,10 @@ mat-list-item ::ng-deep .highlight {
     color: var(--wave-accent-color, grey);
     text-decoration: underline;
 }
+.highlight {
+    color: var(--wave-accent-color, grey);
+    text-decoration: underline;
+}
```

Comment:
This code is subsumed by 
```
.highlight,
mat-list-item ::ng-deep .highlight {
    color: var(--wave-accent-color, grey);
    text-decoration: underline;
}
```
from the four lines above, so you should be able to delete it.

Reply:
deleted